docs/agents/sematext-agent/containers/configuration.md

```diff
@@ -0,0 +1,59 @@
+## Configuration file
+
+For fine-tunning Sematext Agent refer to `st-agent.yml` configuration file. You will have to attach a bind mount with file location and pass the `CONFIG_FILE` environment variable that contains the path to the aforementioned configuration file.
+
+The configuration file accepts all options (listed below) in YAML format.
+```yaml
+# Sematext Agent configuration file
+infra-token: YourInfrastructureMonitoringToken
```

Comment:
Maybe better `...InfraAppToken` so it is clear what it actually is. Btw, we have a small inconsistency with camel-casing used here, in some other places we write it as something like YOUR_INFRA_APP_TOKEN_HERE. We should use the same approach.